.github/workflows/fmt.yml

```diff
@@ -10,4 +10,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: fmt
-      run: cargo fmt -- --check
\ No newline at end of file
+      run: cargo fmt -- --check
+
+    - name: build
+      run: cargo build
```

Comment:
Could it make sense to use `cargo check` or `cargo clippy` instead of `cargo build`? I mean, if we don't really use the built lib, we could just make the CI slightly lighter.

Reply:
Yeah maybe `cargo check` is better :)